### PR TITLE
Adds Four New Cyborg Emotes and Cleans Up Existing Ones

### PR DESCRIPTION
**Pull request by SweaterKittens**
## Four New Cyborg Emotes (With Sounds!) ##

I felt as though the existing cyborg emotes didn't have quite enough range to really convey anything beyond simple yes (ping/chime) and no (buzz/buzz2). This adds a new range of options to Cyborg players, to hopefully be able to convey more "feelings" as a Cyborg and result in a more immersive experience.

The new emotes are:
* [Beep/Beeps](https://hydra-media.cursecdn.com/overwatch.gamepedia.com/2/28/Bastion_-_Beeple.ogg) - _Valentine beeps enthusiastically!_
* [Chuckle/Chortle](https://hydra-media.cursecdn.com/overwatch.gamepedia.com/e/e1/Bastion_-_Sh-sh-sh_dwee%21.ogg) - _W.A.T.C.H.D.O.G. chortles._
* [?/What](https://hydra-media.cursecdn.com/overwatch.gamepedia.com/9/9e/Bastion_-_Zwee.ogg) - _ABDUCTOR queries at Chet Manley._
* [Okay/Alright](https://hydra-media.cursecdn.com/overwatch.gamepedia.com/6/6a/Bastion_-_Doo-woo.ogg) _A.L.I.C.E. acknowledges Jayne Hightower._

## Cleanup of existing Cyborg emotes ##

There was a duplicate emote, as *ping had two identical entries for no particular reason, one of which was removed. Some of the other emotes did not have a parameter message, that is to say, you could *boop, but not *boop a specific person. That of course was literally unplayable. Now most Cyborg emotes should have a parameter option as well.

:cl: Sweaterkittens
add: An additional TB of data has been added to new production Cyborg personality modules. This space has been well-utilized by four new emotes complete with sounds, accessible by using *beep/*beeps, *chuckle/*chortle, *?/*what, and *okay/*alright.
soundadd: Added four new sounds to go with the new silicon emotes: Cyborg-emote-chipper.ogg, Cyborg-emote-chuckle.ogg, Cyborg-emote-confused.ogg, and Cyborg-emote-okay.ogg.
/:cl:

Some of you may recognize where these sounds are from! They are licensed under Creative Commons CC BY-NC-SA 3.0, available [here](https://creativecommons.org/licenses/by-nc-sa/3.0/legalcode).

### CODE REVIEW

**Comment by RandomMarine** [comment]
It's skeleton action figures all over again.

**Comment by cedarbridge** [comment]
+1 for shoutout to my silicon name

**Comment by coiax** [comment]
>It's skeleton action figures all over again.

how

File: code/modules/mob/living/silicon/robot/emote.dm [hunk 1]
```diff
@@ -10,6 +10,7 @@
 	key = "boop"
 	key_third_person = "boops"
 	message = "boops."
+	message_param = "boops %t"
```
Comment (lzimann) [inline code/modules/mob/living/silicon/robot/emote.dm]: Add a period(or exclamation mark, whatever you prefer) to them, please.

**Comment by RandomMarine** [comment]
@coiax sounds ripped straight out of undertale vs sounds ripped straight out of overwatch.

**Comment by RemieRichards** [comment]
@SweaterKittens 
> Some of you may recognize where these sounds are from! They are licensed under Creative Commons CC BY-NC-SA 3.0, available here.

You must put these cyborg emotes in their own folder with a txt that explains their license is CC BY-NC-SA 3.0, and include a link to that license, due to our current sounds/icons being CC BY-SA license.

**Comment by PKPenguin321** [comment]
I sound add sounds to human emotes

File: code/modules/mob/living/silicon/robot/emote.dm (hunk 1 above)
Comment (SweaterKittens) [inline code/modules/mob/living/silicon/robot/emote.dm]: Good eye, thank you!

**Comment by SweaterKittens** [comment]
@RemieRichards 
Got it, thanks Remie! They now have their own folder with a brief explanation of what the Creative Commons license is, and a link to the full one.

**Comment by RemieRichards** [comment]
@SweaterKittens you need to commit the folder :P

**Comment by SweaterKittens** [comment]
@RemieRichards 
YES THAT DOES HELP. I'm not sure exactly what happened, it was in my last commit I thought, but the change was unchecked or something so it didn't go through.

**Comment by Incoming5643** [comment]
I would like better assurances that these files are actually CC BY-NC-SA 3.0 and not something blizzard made themselves that is almost certainly proprietary.

Also on a totally subjective level I just think references to very recent games that may or may not end up becoming memorable are tacky. Nothing personal though.

**Comment by SweaterKittens** [comment]
@Incoming5643 
It was not meant to be a reference. It was merely a happy coincidence that I was looking for some quality new Cyborg sounds and found these which, in my opinion, sound great in a vacuum whether or not you know their origins. 

They appeared to be available under CC, but since that's been brought into question this doesn't appear as though it's going to happen anyway.

**Comment by kevinz000** [comment]
?!

**Comment by Haevacht** [comment]
http://www.beepbox.co/#5s6k7l00e03tba7g0fj7i0r1w4111f2000d3111c0000h3000v0000o3210b4h4h4h4h4h4h4h4h4h4h4h4h4h4h4h4h4h4h4h4h4h4p1xINO6F5aKSotTjenrS7sEEKotrhD000000

Beepbox is the shit lad.

**Comment by coiax** [comment]
I think the old sounds are fine. And let's be honest here, it's not like we've suddenly started caring about copyright. *COUGH* The Room intro lobby music *COUGH*

**Pull request closed without merging**